### PR TITLE
Clean code in base.html #17

### PR DESCRIPTION
**Pull request by rajujha373**
created a base.html file which can be included wherever necessary

### CODE REVIEW

File: home/templates/home.html [hunk 1]
```diff
@@ -1,37 +1,12 @@
...
-  </div>
-</nav>
-
-  </body>
-</html>
+{% extends 'base.html' %}
+
+{% load static %}
+
+{% block styles %}
```
Comment (divyam3897) [inline home/templates/home.html]: Add all the styles in a seperate .css file.

Comment (rajujha373) [reply home/templates/home.html]: you have used {% static 'images/background.jpg' %} in the style section. Jinja templating cannot be done inside static files... if you need a separate css file, you need to add it inside the templates folder :) shall i do it like that?

Comment (divyam3897) [reply home/templates/home.html]: sure, we add the static files to staticFiles, you can create a css folder in it and modify the settings accordingly.

**Comment by divyam3897** [comment]
It has been fixed in https://github.com/pclubuiet/website/commit/cd17bb69e0efc48c583e3c08d69f35b951855a18 Thanks

**Pull request closed without merging**